Applet/src/com/android/javacard/keymaster/KMKeymintDataStore.java

```diff
@@ -92,6 +93,8 @@ public class KMKeymintDataStore implements KMUpgradable {
   private byte[] challenge;
   // Secure Boot Mode
   public byte secureBootMode;
+  // Flag to store KeyMint upgrade from 2-3
+  public boolean kmUpgrade200To300;
```

Comment:
It is a good idea to use "short previousPackageVersion" so that this will be helpful in future.